coding-standards/flutter.md

```diff
@@ -1,9 +1,34 @@
 # Flutter Coding Standards
 
-## Coding Style Guide
-
+## Flutter Coding Style Guidelines
+We adapted standards to design and program Flutter, from high-level architectural principles all the way to indentation rules, from Flutter offical git repo :</br>
 https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
+## Dart Coding Style Guidelines
+Dart Style Guide focuses primarily on Dart-specific conventions: https://dart.dev/effective-dart
```

Comment:
Is dart different from flutter?

Reply:
Flutter is UI sdk comes with widgets and used for UI purposes, dart is client-side programming language use in flutter app development